lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_version_helper.rb

```diff
@@ -180,6 +180,14 @@ def self.get_internal_version
           get_version_strings()[1]
         end
 
+        # Returns the current value of the `BUILD_NUMBER` key from the public xcconfig file
+        #
+        # @return [String] The current build number according to the public xcconfig file.
+        #
+        def self.get_build_number
+          read_build_number_from_config_file(ENV['PUBLIC_CONFIG_FILE'])
```

Comment:
Have you considered passing the file as an argument? 

See discussion in #380 and the work done in #445.

Reply:
@mokagio Ready for re-review. It now uses an argument instead of the env variable